test/TEST-03-JOBS/TEST-03-JOBS.units/shortcut-restart.service

```diff
@@ -0,0 +1,7 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+[Service]
+Type=oneshot
+ExecStartPre=sleep 10
```

Comment:
Can we use `systemd-notify` to get rid of this sleep? (Make the service `Type=notify`?)